en/docs/install-and-setup/setup/distributed-deployment/synchronizing-artifacts-in-a-gateway-cluster.md

```diff
@@ -147,3 +153,34 @@ password = "wso2carbon"
 
 Add the tables of the `AM_GW_API_ARTIFACTS` and `AM_GW_PUBLISHED_API_DETAILS` to this new database that you are specifying. 
 The scripts to create these tables are in the `<API-M_HOME>/dbscripts/apimgt/` directory.
+
+
+## FAQ About Inbuilt Artifact Synchronizer
+
+### In this architecture, does the Gateway requires a database connection?
+No. Gateway does not need to connect with the database to pull the artifacts. It requests from the Traffic manager and traffic manager connects with the database. In this architecture
+traffic manager plays a vital roles and maintain the connections with the database.
+
+### If 1st request (i.e saving artifacts to DB) failed and 2nd request (i.e events to TM) success, what will happen? Is this working as a transactional base? (optimistic or pessimistic)
```

Comment:
Shall we make all questions bold and remove the heading tags? Otherwise this gets listed in the right side menu as well and becomes difficult to read.

Reply:
Fixed with 1269526